SchoolProject/src/main/java/commands/TeacherReciver.java

```diff
@@ -0,0 +1,41 @@
+package commands;
+
+import helpers.Grade;
+import models.Student;
+import repositories.StudentRepository;
+
+public class TeacherReciver {
```

Comment:
Looks like this class is a service. Do you remember the latest example we did with unit testing? Maybe you can rename this class as TeacherService and add it into the corresponding package

Reply:
Although... the objective for this class is to perform the operation... got it!, so you can keep it like it is...